examples/offline_inference_vision_language.py

```diff
@@ -28,7 +28,7 @@ def run_aria(question: str, modality: str):
               tokenizer_mode="slow",
               trust_remote_code=True,
               dtype="bfloat16",
-              mm_cache_preprocessor=args.mm_cache_preprocessor)
+              disable_mm_cache_preprocessor=args.disable_mm_cache_preprocessor)
```

Comment:
Just a very small nit: do you know it makes sense to rename this arg to `disable_mm_preprocessor_cache`? I think the naming is more intuitive in that way

Reply:
yeah, will rename to disable_mm_preprocessor_cache